### PR TITLE
[Uni merge 🦄 ] Switch network adaptation

### PR DESCRIPTION
**Pull request by W3stside**
WIP - adds network switching via uni merge

THIS WILL CHANGE WHEN WE RE-MERGE UNI V4.14+ (UNI is currently at v4.19.x)

### CODE REVIEW

**Comment by alfetopito** [comment]
Github! Why you no deploy? o.O

File: src/custom/api/paraswap/index.ts [hunk 1]
```diff
@@ -19,8 +19,6 @@ function getParaswapChainId(chainId: ChainId): NetworkID | null {
     // Only Mainnnet and Ropsten supported
     //  See https://developers.paraswap.network/api/list-all-tokens
     case ChainId.MAINNET:
-    case ChainId.ROPSTEN:
-      return chainId
```
Comment (alfetopito) [inline src/custom/api/paraswap/index.ts:23]: Aren't you removing too much?

`return chainId` should stay, I think

**Comment by W3stside** [comment]
> Btw, @W3stside , I have created separate issues for 2 and 11 cases, but I'm not sure whether I need to create separate issues for 12-14 cases, or they are fixed there. Could you please clarify?

2,9,11 i think should be logged as issues

**Comment by elena-zh** [comment]
One more question related to networks dropdow to a not connected user. 
When I press on any network, I call the connection modal. And I expect to be connected to the selected network. 
But as a result I appear to be connected to the network that is selected in my MM. 
Can we force network changing in the MM to the required one after connection?
See the video for more details: https://watch.screencastify.com/v/Aamruv5B3nzgYpnD0jtt

**Review by fairlighteth — APPROVED** [review]
Approving to keep this going asap. I see some stylistic smaller issues but can address this separately 👍🏼

**Comment by elena-zh** [comment]
Issue with menu scroll in mobile: #1556 
Issue with Twitter action in the menu #1557

**Review by elena-zh — APPROVED** [review]

**Pull request merged**